duckduckgo/duckduckgo-autofill
# Only show in-context signup if recently installed

**Reviewer:** @GioSensation @shakyShane 
**Asana:** https://app.asana.com/0/0/1204038361216232/f

## Description
Only show in-context signup if the extension indicates it's been recently installed

## Steps to test
1. Install [extension branch `abrown/changes-to-in-context-signup-treatment`](https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/abrown/changes-to-in-context-signup-treatment?expand=1) with this branch of autofill
2. Install extension and confirm in-context signup shows
3. Change `isInstalledWithinDays` function in extension branch to use `v366-1` instead of `settings.getSetting('atb')`
   - Confirm in-context signup no longer shows
